llvm/include/llvm/Support/GenericLoopInfo.h

```diff
@@ -294,6 +294,10 @@ template <class BlockT, class LoopT> class LoopBase {
   /// Otherwise return null.
   BlockT *getUniqueExitBlock() const;
 
+  /// Return the exit block for the latch if one exists. This function assumes
+  /// the loop has a latch.
```

Comment:
I think all loops have at least one latch (a branch back to the header of the loop). Perhaps 'This function assumes the loop has a single latch.'?